restart_process/release.sh

```diff
@@ -2,10 +2,13 @@
 
 set -e
 
-TIMESTAMP=$(date +'%Y-%d-%m')
+TIMESTAMP=$(date +'%Y-%m-%d')
 IMAGE_NAME='tiltdev/entr'
 IMAGE_WITH_TAG=$IMAGE_NAME:$TIMESTAMP
 
+# build our binary
+env GOOS=linux GOARCH=amd64 go build -o tilt-restart-wrapper main.go
```

Comment:
I'd expect this to need to be a static binary as well. Have you tested it on `scratch`?

Reply:
i... thiiink it is static? see: https://www.reddit.com/r/golang/comments/53nl4b/why_all_go_binaries_are_staticallylinked/

Also, it works on `scratch`. (Live update does NOT work on `scratch` but that's a different kettle o fish 😅 )